[IMP] sales: pricelists ecommerce (#12532)
odoo/documentation: content/applications/sales/sales/products_prices/prices/pricing.rst

> .. note::

Reviewer: This is a nitpick, but in general I don't think it is the best practice to have three admonition blocks in a row (especially when two are the same type). It isn't a definite "don't" but if it is possible to rearrange this section a bit, I think it is worth it.

Reviewer 2: Thank you for this tip! Implemented the suggestion and will pocket for future docs

If working in a multinational company, select the countries where this pricelist will apply under
the :guilabel:`Country Groups` column.

.. warning::

>    When a country group is assigned to a pricelist, country groups must be assigned to *all*

Reviewer: ```suggestion
   When a country group is assigned to at least one pricelist, country groups must be assigned to *all*
```

>    pricelists in the database. This is because when country groups are assigned to at least one
>    pricelist, Odoo will check the visitor’s IP address and apply the matching country group for

Reviewer: ```suggestion
   pricelists in the database. This is because once a country group is assigned to a
   pricelist, Odoo checks the visitor's IP address and apply the matching country group for
```

Under the :guilabel:`Ecommerce` tab, price rules can be configured for products sold on an
:doc:`Ecommerce website <../../../../websites/ecommerce/products>`.

With pricelists, online stores can:

- Display different prices for logged-in users versus guest visitors.
- Offer region-specific pricing by assigning country groups.
- Enable promotional discounts and loyalty-based pricing.
- Allow customers to select from multiple pricing options.

By leveraging pricelists, eCommerce businesses can implement strategic pricing models that drive
sales while maintaining full control over pricing rules and promotions.

> 

Reviewer 2: Did you mean the intro of the doc? I'm thinking to keep it here in order to preserve the intention of the whole doc as Sales app documentation, and to keep eCommerce info in the eCommerce section.

To enable the pricelist to be visible, select the target website in the :guilabel:`Website` field.

.. important::
   A pricelist is a website default for public users (users not signed in to Odoo accounts) only
   once they are assigned to the website, and if they are not assigned a country group.

The :guilabel:`Selectable` can be enabled to allow the customer to choose this pricelist when
shopping on the website.

.. important::
   The :guilabel:`Selectable` column is only applicable to Odoo **eCommerce**. This option allows

>    website visitors to choose a pricelist when shopping in the **eCommerce** website.

Reviewer 2: Removed redundant sentence

   If one pricelist is set as selectable, that pricelist is applied as the default pricelist for
   public users, and no selectable option is available for the user.

>    If two or more pricelists are set as selectable, the first pricelist in the sequence on the

Reviewer 2: Based on my conversation with you after I first spoke with the SME, the idea was that because this configuration is done from the Sales app, it could be justified to fit in the Sales documentation. It is also needed here as it's not currently captured in the eCommerce documentation, so users do not have access to this info at the moment. 

If this is something that has changed I'm happy to revise, will defer to you.

Under the :guilabel:`Ecommerce` tab, price rules can be configured for products sold on an
:doc:`Ecommerce website <../../../../websites/ecommerce/products>`.

With pricelists, online stores can:

- Display different prices for logged-in users versus guest visitors.
- Offer region-specific pricing by assigning country groups.
- Enable promotional discounts and loyalty-based pricing.
- Allow customers to select from multiple pricing options.

By leveraging pricelists, eCommerce businesses can implement strategic pricing models that drive
sales while maintaining full control over pricing rules and promotions.

To enable the pricelist to be visible, select the target website in the :guilabel:`Website` field.

.. important::
   A pricelist is a website default for public users (users not signed in to Odoo accounts) only
   once they are assigned to the website, and if they are not assigned a country group.

The :guilabel:`Selectable` can be enabled to allow the customer to choose this pricelist when
shopping on the website.

.. important::
   If one pricelist is set as selectable, that pricelist is applied as the default pricelist for

>    public users, and there will be no selectable option for the user.

Reviewer: ```suggestion
   public users, and no selectable option is available for the user.
```
